data/garden/reference/countries_regions.csv

```diff
@@ -226,7 +226,7 @@ SLB,Solomon Islands,SB,SLB,813.0,SOL,940.0,SOI,BP,,SLB,SLB,http://www.wikidata.o
 SOM,Somalia,SO,SOM,726.0,SOM,520.0,SOM,SO,SMLA,SOM,SOM,http://www.wikidata.org/entity/Q1045,Somalia,82.0,747.0,,
 ZAF,South Africa,ZA,ZAF,199.0,SAF,560.0,SAF,SA,SAFR,ZAF,ZAF,http://www.wikidata.org/entity/Q258,South Africa,81.0,788.0,,
 SGS,South Georgia and the South Sandwich Islands,GS,SGS,,,,,,,,,http://www.wikidata.org/entity/Q35086,South Georgia and the South Sandwich Islands,352.0,807.0,,
-KOR,South Korea,KR,KOR,542.0,ROK,732.0,ROK,KO,KORS,KOR,KOR,http://www.wikidata.org/entity/Q884,South Korea,127.0,673.0,,"[""Korea, Rep."", ""Korea, Republic Of"", ""Republic of Korea""]"
+KOR,South Korea,KR,KOR,542.0,ROK,732.0,ROK,KO,KORS,KOR,KOR,http://www.wikidata.org/entity/Q884,South Korea,127.0,673.0,,"[""Korea"", ""Korea, Rep."", ""Korea, Republic Of"", ""Republic of Korea""]"
```

Comment:
I'm not sure this is a good alias. "Korea" does not mean "South Korea", even if for this particular dataset it may mean that.

Reply:
good point